samples/newChat/css/dashboard.css

```diff
@@ -768,13 +614,18 @@
     color: #778594;
 }
 
-.create_dialog_form {
+.create_dialog_form,
+.update_dialog_form {
```

Comment:
Are you can use one selector with modifier for form if you form has equals style.